awscli/examples/networkflowmonitor/untag-resource.rst

```diff
@@ -0,0 +1,9 @@
+**To remove a tag to the specified resource**
+
+The following ``untag-resource`` example removes a tag to the monitor in the specified account. ::
+
+    aws networkflowmonitor untag-resource \
+        --resource-arn arn:aws:networkflowmonitor:us-east-1:123456789012:monitor/Demo \
+        --tag-keys stack  
+
+This command produces no output.
```

Comment:
Every example needs a for more information link.
https://aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link